doc/conf.py

```diff
@@ -390,6 +398,13 @@
 }
 breathe_default_project = "Zephyr"
 
+# Changing breathe configuration to force "c" domain doesn't work
```

Comment:
Commented out code; should this be removed?

Reply:
I left the comment in as a reminder not to try using the "c" domain for breathe (until breathe support if upgraded).